app/Http/Controllers/AdminController.php

```diff
@@ -80,4 +80,24 @@ public function anySetTime()
     dd('test');
   }
 
+  public function sendsms(Request $request){
+
+    $input = Input::all();
+
+    try{
+      Nexmo::message()->send([
+           'to' => '6'.$input['number'],
+           'from' => 'Saimedic Clinic',
+           'text' => 'Updates from Saimedic Clinic : '.$input['text']
```

Comment:
Sorry, I'm not going to pull in changes that include content specific to your business. You'll have to find a way to abstract it if you want it pulled in.